server/base/src/main/java/org/apache/accumulo/server/manager/state/ManagerTabletInfoIterator.java

```diff
@@ -0,0 +1,434 @@
+import org.apache.accumulo.core.iterators.user.WholeRowIterator;
+import org.apache.accumulo.core.manager.state.ManagerTabletInfo;
+import org.apache.accumulo.core.manager.state.ManagerTabletInfo.ManagementAction;
+import org.apache.accumulo.core.manager.thrift.ManagerState;
+import org.apache.accumulo.core.metadata.TServerInstance;
+import org.apache.accumulo.core.metadata.TabletState;
+import org.apache.accumulo.core.metadata.schema.DataFileValue;
+import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.ChoppedColumnFamily;
+import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.CurrentLocationColumnFamily;
+import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.FutureLocationColumnFamily;
+import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.HostingColumnFamily;
+import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.LastLocationColumnFamily;
+import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.LogColumnFamily;
+import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.SuspendLocationColumn;
+import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.TabletColumnFamily;
+import org.apache.accumulo.core.metadata.schema.TabletMetadata;
+import org.apache.accumulo.core.metadata.schema.TabletMetadata.Location;
+import org.apache.accumulo.core.metadata.schema.TabletMetadata.LocationType;
+import org.apache.accumulo.core.util.AddressUtil;
+import org.apache.hadoop.io.DataInputBuffer;
+import org.apache.hadoop.io.DataOutputBuffer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Sets;
+
+public class ManagerTabletInfoIterator extends SkippingIterator {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ManagerTabletInfoIterator.class);
+
+  private static final String SERVERS_OPTION = "servers";
+  private static final String TABLES_OPTION = "tables";
+  private static final String MERGES_OPTION = "merges";
+  private static final String DEBUG_OPTION = "debug";
+  private static final String MIGRATIONS_OPTION = "migrations";
+  private static final String MANAGER_STATE_OPTION = "managerState";
+  private static final String SHUTTING_DOWN_OPTION = "shuttingDown";
+
+  private static void setCurrentServers(final IteratorSetting cfg,
+      final Set<TServerInstance> goodServers) {
+    if (goodServers != null) {
+      List<String> servers = new ArrayList<>();
+      for (TServerInstance server : goodServers) {
+        servers.add(server.getHostPortSession());
+      }
+      cfg.addOption(SERVERS_OPTION, Joiner.on(",").join(servers));
+    }
+  }
+
+  private static void setOnlineTables(final IteratorSetting cfg, final Set<TableId> onlineTables) {
+    if (onlineTables != null) {
+      cfg.addOption(TABLES_OPTION, Joiner.on(",").join(onlineTables));
+    }
```

Comment:
```suggestion
    cfg.addOption(TABLES_OPTION, Joiner.on(",").join(onlineTables));
```

I looked around in the code for things that implement CurrentState and I did not see anything passing null for this for current servers.  The following is the code I looked at in the manager that gets these sets.

https://github.com/apache/accumulo/blob/ce54fb3903749d30b6ccb92038280dfb160dcbb9/server/base/src/main/java/org/apache/accumulo/server/manager/LiveTServerSet.java#L392

https://github.com/apache/accumulo/blob/ce54fb3903749d30b6ccb92038280dfb160dcbb9/server/manager/src/main/java/org/apache/accumulo/manager/Manager.java#L1661-L1687